File: python/2129.py
class Solution:
    def capitalizeTitle(self, title: str) -> str:
        words = title.split()
        for i, word in enumerate(words):
            words[i] = word.lower()
            if len(word) > 2:
                words[i] = word.title()
        return " ".join(words)

File: go/2129.go
package main

import "strings"

func capitalizeTitle(title string) string {
	words := strings.Fields(title)
	for i, word := range words {
		words[i] = strings.ToLower(word)
		if len(word) > 2 {
			words[i] = strings.Title(words[i])
		}
	}
	return strings.Join(words, " ")
}
